powerhome/playbook
# Issue 402 - Add checkbox list to docs

#### Screens

RAILS:
![Screen Shot 2020-07-31 at 11 08 09 AM](https://user-images.githubusercontent.com/3247447/89054315-3337dc80-d31e-11ea-928c-7dfa7eeb9a59.png)
![Screen Shot 2020-07-31 at 12 43 42 PM](https://user-images.githubusercontent.com/3247447/89062116-7f3d4e00-d32b-11ea-9890-d559c6377b15.png)

![Screen Shot 2020-07-31 at 11 09 12 AM](https://user-images.githubusercontent.com/3247447/89054365-49de3380-d31e-11ea-9244-d5701c733597.png)
![Screen Shot 2020-07-31 at 12 44 11 PM](https://user-images.githubusercontent.com/3247447/89062147-8cf2d380-d32b-11ea-8e77-b411f056d405.png)

REACT:
![Screen Shot 2020-07-31 at 11 10 13 AM](https://user-images.githubusercontent.com/3247447/89054452-6da17980-d31e-11ea-8497-fc4eade57bca.png)
![Screen Shot 2020-07-31 at 12 44 54 PM](https://user-images.githubusercontent.com/3247447/89062201-a72cb180-d32b-11ea-8f10-2672c4873cbc.png)

![Screen Shot 2020-07-31 at 11 11 08 AM](https://user-images.githubusercontent.com/3247447/89054515-8e69cf00-d31e-11ea-9585-6c9784dde0c2.png)
![Screen Shot 2020-07-31 at 12 45 28 PM](https://user-images.githubusercontent.com/3247447/89062245-bad81800-d32b-11ea-9178-72609ed0b12d.png)

#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Runway Ticket URL
[https://github.com/powerhome/playbook/issues/402](url)

#### How to test this
1. Go to Playbook
2. Click on "List" in the sidebar navigation
3. Scroll down to the "Checkbox List" card
4. See a list of check boxes
5. Click the entire row for each checkbox to see if it gets checked or unchecked
6. Test these steps for the dark mode and for the light and dark mode for the React code

#### Checklist:
